pages/[owner]/[repo]/index.tsx

```diff
@@ -24,19 +25,26 @@ interface StatsCardProps {
 }
 const ChildWithSWR = (props: { owner: string; repo: string }) => {
   const { owner, repo } = props;
-  const { data, error, mutate } = useSWR<PaginatedDataResponse, Error>(`prs/search?repo=${owner}%2F${repo}`);
+
+  const { data: openedPrs } = useContributorData(`${owner}/${repo}`, 0, "open");
+  const { data: prevMonthOpenedPrs, meta: prevMonthOpenedPrsMeta } = useContributorData(`${owner}/${repo}`, 30, "open");
+
+  const { data: closedPrs } = useContributorData(`${owner}/${repo}`, 0, "closed");
+  const { data: prevMonthClosedPrs, meta: prevMonthClosedPrsMeta } = useContributorData(
+    `${owner}/${repo}`,
+    30,
+    "closed"
+  );
+
+  const { data: currentData, isError, isLoading, meta } = useContributorData(`${owner}/${repo}`);
+
   // fetch previous months data seperately to compare
-  const {
-    data: prevMonthData,
-    error: prevMonthError,
-    mutate: prevMonthMutate,
-  } = useSWR<PaginatedDataResponse, Error>(`prs/search?repo=${owner}%2F${repo}&prev_days_start_date=30`);
 
-  if (!data && !error) {
```

Comment:
This change is breaking the build.

Reply:
nvm. I figure out that prCounts file is duplicated and I was looking at the original, but there is one in the lib.